### ISSUE TITLE
Blinky demo does not compile

### ISSUE THREAD

**Issue by chrta** [kvasir-io/Kvasir#50]
I use revision 729fa5cd497c7002b34255bacbdd08ff2ca16df4 and the LPC1549 in the current lpcxpresso 8.0.
I get lots of compile errors (>80), e.g.
Kvasir/Lib/StartUp/StartUp.hpp:111:17: error: 'FlattenT' is not a member of 'Kvasir::MPL'

**Comment by odinthenerd** [comment]
Yes we have been neglecting our demos. The focus of kvasir has been shifting from a lib for a few chips which does a lot to a lib that works on every cortex meant to co exist with other libraries. For the upcoming version 1.0 we are focused on exhaustive  SFR abstraction functionality. I'm on vacation right now but I will get you a working demo next week. I have the same board.
